apps/src/lib/levelbuilder/lesson-editor/ResourcesEditor.jsx

```diff
@@ -60,6 +61,9 @@ class ResourcesEditor extends Component {
     resourceContext: PropTypes.string.isRequired,
     resources: PropTypes.arrayOf(resourceShape).isRequired,
 
+    addRollupsType: PropTypes.oneOf(['course', 'script']),
+    addRollupsId: PropTypes.string,
```

Comment:
Is addRollupsId the script or course name? Is so I'm wondering if maybe that could be a little more clear with the name

Reply:
It could be either. I'm happy to break it up into two different props and definitely open to a different name.

I could also just combine these and pass in the url, as `addRollupsUrl` or something, which would be constructed by the Script and Course edit pages.

Reply:
I kind of like like the idea of combining them. What do you think?

Reply:
Yeah the idea is definitely growing on me. I'll update